templates/.circleci/config.yml.tpl

```diff
@@ -27,6 +27,9 @@ contexts: &contexts
 {{- else }}
   {{- /* Always generate an array if there are no contexts */}}
   []
+{{- end }}
+{{- if (stencil.Arg "notifications.releaseFailureSlackChannel") }}
+  - tray-webhooks
```

Comment:
Instead of setting the context name here, it'd be better to put it in our box config "extra-contexts" config. Then users will get it automatically, while still allowing OSS peeps to store these however they want.

https://github.com/getoutreach/box/blob/main/box.yaml#L17

Reply:
Great idea! move https://github.com/getoutreach/box/pull/16